docs/user/Build.md

```diff
@@ -0,0 +1,93 @@
+# Build OpenROAD
+
+## Install dependencies
+
+To install dependencies to develop or build and run OpenROAD locally,
+follow [this procedure](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html).
+
+> WARNING
+>
+> `etc/DependencyInstaller.sh` is an implementation detail of
+> the setup procedures and can be harmful if invoked incorrectly.
+> Do not invoke it directly.
```

Comment:
What is this supposed to mean? How are users supposed to install the dependencies then?

Reply:
This is redirect to OpenROAD-flow-scripts. If developers want to install only OpenROAD mean how do they install dependencies? I am also having same question.
I've been discussing with Vitor and Matt on this. 
Once we get right solution, this part will be removed.

Reply:
As is, it's just a scary warning without telling users what to actually do. I would much prefer something like this:
```suggestion
> :warning: Warning :warning:
>
> `etc/DependencyInstaller.sh` defaults to installing system packages and requires sudo access.
> These packages can affect your environment. We recommend users install dependencies locally using
> `setup.sh` from OpenROAD-flow-scripts
```
Eventually, there should be some actual documentation about the dependency installer because as is, it's rather confusing about which scripts are supposed to run.